Run Cypress in CI with mocks. (#269)
FutureNHS/futurenhs-platform: frontend/README.md

Open [http://localhost:3000](http://localhost:3000) with your browser to see the result.

You can start editing the page by modifying `pages/index.js`. The page auto-updates as you edit the file.

## Testing

### Cypress

> Cypress tests for chrome and firefox currently run as part of our pipeline. It's also possible to run in a local browser before pushing code.

Reviewer: ```suggestion
Cypress tests run as part of our CI pipeline. It's also possible to run in a local browser before pushing code.
```
